config/docker_auth/auth_config.yml

```diff
@@ -0,0 +1,27 @@
+server:
```

Comment:
Can you put a comment at the top of this file that links to https://github.com/cesanta/docker_auth/blob/main/examples/reference.yml so we can find the docs quickly in the future?

Reply:
sure